### PR TITLE
Update decidim to version 0.9.3 and fix scopes

### PR DESCRIPTION
**Pull request by isaacmg410**

### CODE REVIEW

File: app/views/decidim/sabarca/admin/mayor_neighborhoods/_form.html.erb [hunk 1]
```diff
@@ -38,7 +38,9 @@
 </div>
 <div class="row" >
   <div class="columns xlarge-6" >
```
Comment (agustibr) [inline app/views/decidim/sabarca/admin/mayor_neighborhoods/_form.html.erb]: @isaacmg410 please remove unused `<div>`

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <% end %>
 
 <%= render partial: 'pages/home/extended' %>
-<%= render partial: 'pages/home/highlighted_processes' %>
+<%#= render partial: 'pages/home/highlighted_processes' %>
```
Comment (agustibr) [inline app/views/pages/home.html.erb]: @isaacmg410 wtf?

File: app/views/decidim/sabarca/admin/mayor_neighborhoods/_form.html.erb (hunk 1 above)
Comment (isaacmg410) [reply app/views/decidim/sabarca/admin/mayor_neighborhoods/_form.html.erb]: @agustibr done! 😃

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (isaacmg410) [reply app/views/pages/home.html.erb]: @agustibr done! 😃

**Pull request merged**